Week03/lesson05/homework/week3-relaxr-part2/css/style.css

```diff
@@ -0,0 +1,223 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
```

Comment:
I realise the `/* IE */` comment was there in some old code I sent you, so this is totally on me not you, but remove it in future ok? It's misleading as that code is applicable to all browsers, not just Internet Explorer. Thanks! 😸